sympy/stats/rv.py

```diff
@@ -796,7 +800,11 @@ def probability(condition, given_condition=None, numsamples=None,
     from sympy.stats.symbolic_probability import Probability
     if evaluate:
         return Probability(condition, given_condition).doit(**kwargs)
-    return Probability(condition, given_condition).rewrite(Integral) # will return Sum in case of discrete RV
+    message = ("Using `evaluate=False` returns `Probability` object since version 1.7. "
+              "If you want unevaluated Integral/Sum use "
+              "`P(condition, given_condition, evaluate=False).rewrite(Integral)`")
```

Comment:
```suggestion
    message = ("Since version version 1.7, using `evaluate=False` returns `Probability` object. "
              "If you want unevaluated Integral/Sum use "
              "`P(condition, given_condition, evaluate=False).rewrite(Integral)`")
```